spring-cloud-bus/src/main/java/org/springframework/cloud/bus/jackson/BusJacksonAutoConfiguration.java

```diff
@@ -39,23 +39,20 @@
 public class BusJacksonAutoConfiguration {
 
 	@Bean
+	@ConditionalOnMissingBean(name="busJsonConverter")
```

Comment:
Do you really want the name or just a type? Since it's a pretty specific type, you can remove name.